build/common.project.props

```diff
@@ -124,13 +124,17 @@
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
     <VsixOutputDirName>VS15</VsixOutputDirName>
     <CodeAnalysisRuleSet Condition="'$(CodeAnalysisRuleSet)' == ''">$(BuildCommonDirectory)NuGet.ruleset</CodeAnalysisRuleSet>
-    <EnforceCodeStyleInBuild Condition=" '$(EnforceCodeStyleInBuild)' != '' " >true</EnforceCodeStyleInBuild>
+    <WarningsNotAsErrors>EnableGenerateDocumentationFile</WarningsNotAsErrors>
   </PropertyGroup>
 
   <PropertyGroup>
     <VsixPublishDestination>$(ArtifactRoot)$(VsixOutputDirName)\</VsixPublishDestination>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(EnforceCodeStyleInBuild)' != 'false'">
```

Comment:
`== ''` should have been the pattern. 

Any reason not to update that on line 127?

Many properties are defined there.

Reply:
The documentation on conditions shows that only certain elements support the condition statement.  https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions?view=vs-2022#supported-elements

Reply:
But ya, switched it back to == ''

Reply:
Oh I guess "Property" counts as any property? whatever lol